projects/Generic/linux/linux.x86_64.conf

```diff
@@ -5018,16 +5041,25 @@ CONFIG_CRYPTO_DRBG_HMAC=y
 # CONFIG_CRYPTO_DRBG_CTR is not set
 CONFIG_CRYPTO_DRBG=m
 CONFIG_CRYPTO_JITTERENTROPY=m
-# CONFIG_CRYPTO_USER_API_HASH is not set
-# CONFIG_CRYPTO_USER_API_SKCIPHER is not set
+CONFIG_CRYPTO_USER_API=y
+CONFIG_CRYPTO_USER_API_HASH=y
+CONFIG_CRYPTO_USER_API_SKCIPHER=y
 # CONFIG_CRYPTO_USER_API_RNG is not set
 # CONFIG_CRYPTO_USER_API_AEAD is not set
```

Comment:
Any particular reason why you left `CONFIG_CRYPTO_USER_API_RNG` and `CONFIG_CRYPTO_USER_API_AEAD` not set in Generic, but have enabled both options for RPi and RPi2?